src/main/java/ca/corefacility/bioinformatics/irida/ria/security/LoginSuccessHandler.java

```diff
@@ -1,35 +1,32 @@
 package ca.corefacility.bioinformatics.irida.ria.security;
 
-import ca.corefacility.bioinformatics.irida.model.user.User;
-import ca.corefacility.bioinformatics.irida.repositories.user.UserRepository;
-import com.timgroup.jgravatar.Gravatar;
-import com.timgroup.jgravatar.GravatarDefaultImage;
-import com.timgroup.jgravatar.GravatarRating;
+import java.io.IOException;
+import java.util.Date;
+import java.util.Locale;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
+
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.springframework.security.core.Authentication;
 import org.springframework.security.web.authentication.SavedRequestAwareAuthenticationSuccessHandler;
 import org.springframework.web.servlet.LocaleResolver;
 
-import javax.servlet.ServletException;
-import javax.servlet.http.HttpServletRequest;
-import javax.servlet.http.HttpServletResponse;
-import javax.servlet.http.HttpSession;
-import java.io.IOException;
-import java.util.Date;
-import java.util.Locale;
+import ca.corefacility.bioinformatics.irida.model.user.User;
+import ca.corefacility.bioinformatics.irida.repositories.user.UserRepository;
 
 /**
  * Handles actions for when a user is successfully logged in.
  */
 public class LoginSuccessHandler extends SavedRequestAwareAuthenticationSuccessHandler {
 	private static final Logger logger = LoggerFactory.getLogger(LoginSuccessHandler.class);
 
-	private static final String GRAVATAR_ATTRIBUTE = "gravatar";
-
-	private UserRepository userRepository;
+	private final UserRepository userRepository;
 
-	private LocaleResolver localeResolver;
+	private final LocaleResolver localeResolver;
```

Comment:
Its ok that these are getting set as `final`, but I think your formatter is running amok again.